lib/player.ml

```diff
@@ -0,0 +1,8 @@
+(* Type for each player *)
+type player = { letters : char list }
+
+(** [update_player_list player_lst] gives a player list with the second
+    player becoming the head player. Returns the empty list if empty
+    list is given. *)
+let update_player_list (player_lst : player list) =
```

Comment:
this method name could be clearer. what do you think of something like `advance_player`?